doc/api/inspector.md

```diff
@@ -154,8 +154,9 @@ session.post('Profiler.enable', () => {
     // invoke business logic under measurement here...
 
     // some time later...
-    session.post('Profiler.stop', ({ profile }) => {
+    session.post('Profiler.stop', (err, { profile }) => {
       // write profile to disk, upload, etc.
+      fs.writeFileSync('./profile.cpuprofile', JSON.stringify(profile));
```

Comment:
Should the example use `profile` only if `err` is falsy?

Reply:
Fixed.